mobility/mobility/tools/teleop.cc

```diff
@@ -72,6 +81,7 @@ DEFINE_bool(nocollision, false, "Don't check for collisions during action");
 DEFINE_bool(nobootstrap, false, "Don't move to the starting station on execute");
 DEFINE_bool(noimmediate, false, "Don't execute immediately");
 DEFINE_bool(replan, false, "Enable replanning");
+DEFINE_bool(timesync, false, "Enable time synchronization");
```

Comment:
?

Reply:
I'm not sure how this line disappeared in the ROS2 branch, but it exists in develop and FLAGS_timesync is used down below, so def mandatory...